scripts/xtp_qmmm2qm.in

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+#
+# Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import h5py
+import sys
+import os
+import argparse
+import importlib
+
+VERSION = '@PROJECT_VERSION@ #CSG_GIT_ID#'
+PROGTITLE = 'THE VOTCA::XTP QMMM orbfile converter'
+PROGDESCR = 'Convert QMMM orb file into a DFTGWBSE-like orb file usable by gencube,densityanalysis,...'
+VOTCAHEADER = '''\
+==================================================
+========   VOTCA (http://www.votca.org)   ========
+==================================================
+{progtitle}
+please submit bugs to @PROJECT_CONTACT@
+xtp_qmmm2qm, version {version}
+'''.format(version=VERSION, progtitle=PROGTITLE)
+
+
+parser = argparse.ArgumentParser(prog='xtp_qmmm2qm')
+
+parser.add_argument("-f", "--file", help="Input File (String)")
+parser.add_argument("-r", "--region", help="QMMM region number (Index)")
+parser.add_argument('-v', '--verbosity', action="count", help="Verbose mode")
+args = parser.parse_args()
+
+
+region = "region_" + args.region
+file_name, ext = os.path.splitext(args.file)
+
+
+print (VOTCAHEADER)
+print("...Input file: ", args.file)
+
+print("...Selected region: ", region)
+
+
+# Read input hdf5 file
+data_qmmm = h5py.File(args.file, 'r')
+#Check if selected region exists in the database
+if region in data_qmmm.keys():
+    # Create database similar to the simple QM one
+    new_file_name = file_name + "_QMlike" + ext
+    dataset = h5py.File(new_file_name, "w")
+    data_qmmm.copy(region+'/orbitals', dataset)
+    dataset['QMdata'] = dataset['orbitals']
+    del dataset['orbitals']
```

Comment:
This `del` statement doesn't work because you need to open the file in `append` mode. For more information see [this thread](https://stackoverflow.com/questions/31861724/deleting-hdf5-dataset-using-h5py)

Reply:
I trust your experience but if you naively print(dataset.keys()) before and after the del command you see that the 'orbitals' group is gone. Am I missing something?

Reply:
actually the script should not modify the statefile. It should create a new orbitals file. This would corrupt the statefiles for all other programs and would be unexpected.

Reply:
I have the feeling the (input) statefile is not modified at all. What I do here is copying the 'orbitals' part I need from the input statefile into another statefile (so the original statefile stays untouched).

Reply:
Btw to make the new statefile usable in XTP, we need to change the group name from 'orbitals' to 'QMdata'. I couldn't find a better way to change the name. I simply copy the 'orbitals' group calling it 'QMdata' and then I delete the 'orbitals' group. 
Any suggestion to improve upon this step is more than welcome :)

Reply:
yeah It took my some time to realize that as well. My python is bad. Good concept. :+1:

Reply:
@gtirimbo  the problem is that indeed `del` deletes the `orbitals` group from the HDF5 table (by table I mean the object that stores the groups/dataset input) but it doesn't clean up the memory. The object occupies the same size before and after deleting the dataset.

Reply:
@gtirimbo  I didn't get that you were trying to rename the dataset. You can do that with  with [move](https://docs.h5py.org/en/stable/high/group.html#h5py.Group.move)

Reply:
Ok great! I modified things accordingly to your suggestions